packages/block-library/src/embed/components.js

```diff
@@ -1,139 +1,4 @@
-	const iframeTitle = sprintf( __( 'Embedded content from %s' ), parsedUrl.host );
-	const sandboxClassnames = classnames( type, className, 'wp-block-embed__wrapper' );
-
-	const embedWrapper = 'wp-embed' === type ? (
-		<div
-			className={ sandboxClassnames }
-			dangerouslySetInnerHTML={ { __html: html } }
-		/>
-	) : (
-		<div className="wp-block-embed__wrapper">
-			<SandBox
-				html={ html }
-				scripts={ scripts }
-				title={ iframeTitle }
-				type={ sandboxClassnames }
-			/>
-		</div>
-	);
-
-	return (
-		<figure className={ classnames( className, 'wp-block-embed', { 'is-type-video': 'video' === type } ) }>
-			{ ( cannotPreview ) ? (
-				<Placeholder icon={ <BlockIcon icon={ icon } showColors /> } label={ label }>
-					<p className="components-placeholder__error"><a href={ url }>{ url }</a></p>
-					<p className="components-placeholder__error">{ __( 'Previews for this are unavailable in the editor, sorry!' ) }</p>
-				</Placeholder>
-			) : embedWrapper }
-			{ ( ! RichText.isEmpty( caption ) || isSelected ) && (
-				<RichText
-					tagName="figcaption"
-					placeholder={ __( 'Write caption…' ) }
-					value={ caption }
-					onChange={ onCaptionChange }
-					inlineToolbar
-				/>
-			) }
-		</figure>
-	);
-};
-
-export const EmbedControls = ( props ) => {
-	const { showEditButton, supportsResponsive, allowResponsive, getResponsiveHelp, toggleResponsive, switchBackToURLInput } = props;
-	return (
-		<Fragment>
-			<BlockControls>
-				<Toolbar>
-					{ showEditButton && (
-						<IconButton
-							className="components-toolbar__control"
-							label={ __( 'Edit URL' ) }
-							icon="edit"
-							onClick={ switchBackToURLInput }
-						/>
-					) }
-				</Toolbar>
-			</BlockControls>
-			{ supportsResponsive && (
-				<InspectorControls>
-					<PanelBody title={ __( 'Media Settings' ) } className="blocks-responsive">
-						<ToggleControl
-							label={ __( 'Resize for smaller devices' ) }
-							checked={ allowResponsive }
-							help={ getResponsiveHelp }
-							onChange={ toggleResponsive }
-						/>
-					</PanelBody>
-				</InspectorControls>
-			) }
-		</Fragment>
-	);
-};
-
-export const EmbedEditUrl = ( props ) => {
-	const { icon, label, value, onSubmit, onChange, cannotEmbed } = props;
-	return (
-		<Placeholder icon={ <BlockIcon icon={ icon } showColors /> } label={ label } className="wp-block-embed">
-			<form onSubmit={ onSubmit }>
-				<input
-					type="url"
-					value={ value || '' }
-					className="components-placeholder__input"
-					aria-label={ label }
-					placeholder={ __( 'Enter URL to embed here…' ) }
-					onChange={ onChange } />
-				<Button
-					isLarge
-					type="submit">
-					{ _x( 'Embed', 'button label' ) }
-				</Button>
-				{ cannotEmbed && <p className="components-placeholder__error">{ __( 'Sorry, we could not embed that content.' ) }</p> }
-			</form>
-		</Placeholder>
-	);
-};
+export { default as EmbedControls } from './embed-controls';
```

Comment:
Don't really need this file anymore, since the components can be imported directly from their own files in embed/edit.js